Common/src/main/java/at/petrak/hexcasting/api/casting/eval/sideeffects/OperatorSideEffect.kt

```diff
@@ -18,14 +18,11 @@ import net.minecraft.world.item.ItemStack
  */
 sealed class OperatorSideEffect {
     /** Return whether to cancel all further [OperatorSideEffect] */
-    abstract fun performEffect(harness: CastingVM): Boolean
+    abstract fun performEffect(harness: CastingVM)
```

Comment:
do we want to get rid of early exit entirely? I think I'm fine with errors being all mishap-based but it's possible some effect could want to early exit without loudly failing? I suppose that's less of an argument with the class being sealed. Seems like the sort of thing where if it was unsealed could maybe be useful for some addon somewhere maybe possibly?

Reply:
my interpretation of the SideEffect design is that preconditions should be checked before sideeffect operation, so earlyExit was a weird escape hatch out? I may be wrong here though